src/common/shutdown_state.c

```diff
@@ -208,14 +208,14 @@ shutdown_state_check(struct shutdown_state *curr_sds,
 		 * but there wasn't an ADR failure
 		 */
 		CORE_LOG_WARNING(
-			"The pool was not closed - the ADR failure detection will be reinitialized.");
+			"The ADR failure was detected but the pool was closed properly - reinitializing the ADR failure detection.");
 		shutdown_state_reinit(curr_sds, pool_sds, rep);
 		return 0;
 	}
 	if (dirty == 0) {
 		if (is_uuid_correct)
 			CORE_LOG_WARNING(
-				"The ADR failure was detected but the pool was closed - the ADR failure detection will be reinitialized.");
+				"The ADR failure was detected but the pool was closed properly - the ADR failure detection will be reinitialized.");
```

Comment:
Please apply consistently. xD Instead of:

- the ADR failure detection will be reinitialized x2
- reinitializing the ADR failure detection

please use "reinitializing ADR failure detection". Because:

1. All of these indicate exactly the same outcome.
2. I believe without "the" it sounds a little bit more natural. Sorry for the last minute change.

Reply:
Done.